apple-pie/index.html

```diff
@@ -4,36 +4,58 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- Remember to link your styles -->
+    <link rel="stylesheet" href="styles\style.css" />
   </head>
   <body>
-    Apple Pie
-
-    This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
+    <section class="title">
+      <h1>Apple Pie</h1>
+    </section>
+    
+    <section class="intro">
+    <h3>This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
     be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! 
+    favorites as well!</h3>
+    </section>
+
+    <div>
+      <img class="recipe" src="images\recipe-info.png">
+    </div>
     
-    Ingredients 
+    <hr>
+    <section class="ingredients">
+    <h3>Ingredients</h3>
     
-    1 recipe pastry for a 9 inch double crust pie 
-    1/2 cup unsalted butter 
-    3 tablespoons all-purpose flour
-    1/4 cup water
-    3 tablespoons all-purpose flour
-    1/4 cup water 1/2 cup white sugar
-    1/2 cup packed brown sugar
-    8 Granny Smith apples - peeled, cored and sliced
+   <ul>- 1 recipe pastry for a 9 inch double crust pie </ul> 
+    <ul>- 1/2 cup unsalted butter </ul>
+    <ul>- 3 tablespoons all-purpose flour</ul>
+    <ul>- 1/4 cup water</ul>
+    <ul>- 3 tablespoons all-purpose flour</ul>
+    <ul>- 1/4 cup water 1/2 cup white sugar</ul>
+    <ul>- 1/2 cup packed brown sugar</ul>
+    <ul>- 8 Granny Smith apples - peeled, cored and sliced</ul> 
```

Comment:
La estructura correcta de las listas es <ul> (el padre de todos :D) <li> (cada elemento de la lista! :D)

Te pongo abajo como seria vale?

```suggestion
<ul>
    <li>- 1 recipe pastry for a 9 inch double crust pie </li> 
    <li>- 1/2 cup unsalted butter </li>
    <li>- 3 tablespoons all-purpose flour</li>
    <li>- 1/4 cup water</li>
    <li>- 3 tablespoons all-purpose flour</li>
    <li>- 1/4 cup water 1/2 cup white sugar</li>
    <li>- 1/2 cup packed brown sugar</li>
    <li>- 8 Granny Smith apples - peeled, cored and sliced</li>
</ul>
```